tests/operators/test_python.py

```diff
@@ -396,7 +397,7 @@ def test_without_dag_run(self):
                 elif ti.task_id == 'branch_2':
                     self.assertEqual(ti.state, State.SKIPPED)
                 else:
-                    raise Exception
+                    raise ValueError(f'Invalid task id {ti.task_id} found!')
```

Comment:
What is this change for? It seems unrelated to your change.

Reply:
I was changing this from `Exception` to `ValueError` in one of the new test cases I added following previous suggestions from reviewers. So I replaced the same usage in this test module too. I will revert this change if you prefer leaving the existing code untouched.

Reply:
Cool keep it